cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/web/PublicAffiliationController.java

```diff
@@ -0,0 +1,42 @@
+package edu.asu.diging.cord19.explorer.web;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.cord19.explorer.core.model.Publication;
+import edu.asu.diging.cord19.explorer.core.model.impl.PublicationImpl;
+import edu.asu.diging.cord19.explorer.core.mongo.PublicationRepository;
+
+@Controller
+public class PublicAffiliationController {
+
+    @Autowired
+    private PublicationRepository pubRepo;
+
+    @RequestMapping("/publicAffiliation")
```

Comment:
let's make this `/publications` unless it's used somewhere else

Reply:
actually, this should probably just be the same controller as for the publication browse page, but filter the publication in cases like this.